pennylane/devices/tests/test_gates.py

```diff
@@ -150,6 +204,49 @@
 # ===============================================================
 
 
+class TestSupportedGates:
+    """Test that the device can implement all gates that it claims to support."""
+
+    @pytest.mark.parametrize("operation", all_ops)
+    def test_supported_gates_can_be_implemented(self, device_kwargs, operation):
+        """Test that the device can implement all its supported gates."""
+        device_kwargs["wires"] = 3  # maximum size of current gates
```

Comment:
(Minor suggestion): would it be worth factoring out `3` higher up in this file? Seems to be a constant that appears more than once and could be changed easily later if defined at a single place

Reply:
Yes, and we wanted to keep test data as close to tests as possible, so people can understand what is tested...